config/filter.d/openhab.conf

```diff
@@ -0,0 +1,16 @@
+# Openhab brute force auth filter: /etc/fail2ban/filter.d/openhab.conf:
+#
+# Block IPs trying to auth openhab by web or rest api
+#
+# Matches e.g.
+# 12.34.33.22 -  -  [26/sept./2015:18:04:43 +0200] "GET /openhab.app HTTP/1.1" 401 1382 
+# 175.18.15.10 -  -  [02/sept./2015:00:11:31 +0200] "GET /rest/bindings HTTP/1.1" 401 1384
+
+[Definition] 
+failregex = ^<HOST> -  - \s*\[\]\s* "GET /.*HTTP/1.1" 401 (1382|1384)
```

Comment:
two things:
- please do not use not hard-anchored at end (`$`) regexp together with greedy catch-all (`.*`)
- I believe the last numbers are variable (size of 401-page? url-length? can be changed for future versions), don't match it
- don't restrict request method (GET, POST,...) and protocol (HTTP/1.1), may be a weak spot later;

I think this regexp would be better (not tested):
```regexp
^<HOST>\s+-\s+-\s+\[\]\s+"[A-Z]+ .*" 401 \d+\s*$
```